media/Config/index.js

```diff
@@ -0,0 +1,191 @@
+const oneImport = {
```

Comment:
Maybe add copyright comment?

Reply:
I see. I'll fix it

Reply:
May I ask why the `.css` file is ok without a copyright, but `.js` isn't?

Reply:
> May I ask why the .css file is ok without a copyright, but .js isn't?

I don't know. I didn't think of adding copyright comemnts for that PR.
We can add copyright comments if `.css` files are also required/possible to add comments.

Reply:
https://stackoverflow.com/questions/3762794/css-copyright-issue seems related

Reply:
> https://stackoverflow.com/questions/3762794/css-copyright-issue seems related

Ahh I got it. Then we don't need to add copyrights to `.css` or `.html` files.